org.gnome.FileRoller.json

```diff
@@ -0,0 +1,91 @@
+{
+    "app-id" : "org.gnome.FileRoller",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "file-roller",
+    "rename-icon": "org.gnome.ArchiveManager",
+    "copy-icon": true,
```

Comment:
Hmm would be nice to rename those upstream. Feel free to open an issue regarding that.

Reply:
Will do, but I have the impression that it is more involved than that... the app uses the names FileRoller and ArchiveManager interchangeably.

Reply:
https://gitlab.gnome.org/GNOME/file-roller/-/issues/98